CHANGELOG.md

```diff
@@ -2,8 +2,91 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 43 end 2016-07-11
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+43+Ending+July+11%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Service resolution based on Provision Order
+  - Added /System/Process/MiqEvent instance
+  - Added Provider refresh call to Amazon retire state machine in Pre-Retirement state
+  - Service Dialogs: Added ‘Visible’ flag to all dialog fields
+  - Class Schema allows for EMS, Host, Policy, Provision, Request, Server, Storage, and VM(or Template) datatypes
+    - Value is id of the objects
+    - If object is not found, the attribute is not defined.
+- Platform: Custom attributes
+  - Now selectable as a column
+  - Support in report filters and charts
+- Providers
+  - Amazon: Public Images Filter
+  - Networking
+      - Separate Google Network Manager
+      - NFV: VNFD Templates and VNF Stacks
+  - Pluggability: SupportsFeatureMixin
+      - supports :provisioning
+      - supports : discovery
+  - Hawkular
+    - Deployment entity operations: Undeploy, redeploy
+    - Server operations: Reload, suspend, resume
+    - Live metrics for datasources and transactions
+    - Performance reports for middleware servers
+    - Support for alert profiles and alert automated expressions for middleware server
+    - Crosslink middleware servers with RHEV VMs
+    - Collect and display deployment status
+    - Datasources topology view
+- REST API
+  - Support for report schedules
+  - Support for approving or denying service requests
+  - Support for OpenShift Container Deployments
+  - Support for Virtual Templates
+- Service Broker
+  - Started work on Service Broker to allow ManageIQ to select VM for you based on criteria (cloud, cost, or performance)
+  - Added API backend for Resourceless Servers
+  - Added datastore for the default settings for resourceless servers
+- SmartState: Generalized disk LRU caching module
+  - Caching module can be used by any disk module, eliminating duplication.
+  - Can be inserted “higher” in the IO path.
+  - Configurable caching parameters (memory vs performance)
+  - Will be employed to address Azure performance and throttling issues.
+  - Other disk modules converted over time.
+- User Interface
+  - Settings moved to top right navigation header
+  - C3 Charts fully implemented - chart interaction coming soon!
+  - Tagging for Ansible Tower job templates
+  - Live Search added to bootstrap selects
+  - Self Service UI Order History with detail
+
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+43+Ending+July+11%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Generic Object: Model refactoring/cleanup, use PostgreSQL jsonb column
+  - Null Coalescing Operator
```

Comment:
@mfeifer This is a new attribute data type so should be listed in the "Added" section.